src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py

```diff
@@ -179,62 +165,39 @@ def __parseFormParams(self):
 
         return S_OK((typeName, reportName, start, end, pD, grouping, extraParams))
 
-    @asyncGen
     def web_generatePlot(self):
-        callback = {}
-        retVal = yield self.threadTask(self.__queryForPlot)
-        if retVal["OK"]:
-            callback = {"success": True, "data": retVal["Value"]["plot"]}
-        else:
-            callback = {"success": False, "errors": retVal["Message"]}
-        self.finish(callback)
+        if (retVal := self.__queryForPlot())["OK"]:
+            return {"success": True, "data": retVal["Value"]["plot"]}
```

Comment:
The other returns always use string "true", "false", is that equivalent with the booleans?